File: 151004/Bordilovsky/DC_Kafka&Redis/Publisher/Services/Mappers/IssueMapper.cs
using AutoMapper;
using DC_REST.DTOs.Request;
using DC_REST.DTOs.Response;
using DC_REST.Entities;

namespace DC_REST.Services.Mappers
{
	public class IssueMapper : Profile
	{
		public IssueMapper() 
		{ 
			CreateMap<Issue, IssueResponseTo>();
			CreateMap<IssueRequestTo, Issue>();
		}
	}
}

File: 151002/Samusenko/lab_three/publisher/src/main/java/com/example/publisher/service/mapper/IssueMapper.java
package com.example.publisher.service.mapper;

import com.example.publisher.model.entity.Issue;
import com.example.publisher.model.request.IssueRequestTo;
import com.example.publisher.model.response.IssueResponseTo;
import org.mapstruct.Mapper;

import java.util.List;

@Mapper(componentModel = "spring")
public interface IssueMapper {
    IssueResponseTo getResponse(Issue issue);
    List<IssueResponseTo> getListResponse(Iterable<Issue> issues);
    Issue getIssue(IssueRequestTo issueRequestTo);
}